neon_cli/neon

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import os
+import subprocess
+import sys
+from typing import List
+
+import requests
+
+import logging
+
+# enable debug logging
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+BIN_DIR = os.getenv('NEON_BINARIES_DIR', './')
+
+
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+
+if sys.version_info.major != 3 and sys.version_info.minor < 9:
+    eprint('Warning: This script was written on python 3.9, your version is ')
```

Comment:
it seems that you forgot to add info about user's version

Reply:
added